### PR TITLE
Update/docs/slack connection

### PR DESCRIPTION
**Pull request by ispringle**
Tryna make docs suck less.

### CODE REVIEW

File: docs/BASICS.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+   After you configure the bot it should drop you at the "Basic Information"
+   page. Click "Bot" and it'll jump to the App Home page where you can define
+   the bot's scope and get a token. Legobot is a "classic" app and requires
+   the "Legacy Bot User" scope (also called 'bot' in the Oauth scope page).
+   Click the "Add Legacy Bot User" and fill in the name of the bot.
+4. Go to "Install App" on the sidebar and then click "Install to Workspace"
+   and then "Allow". It will redirect to a page with your token, you want
+   the "Bot User OAuth Token"
+5. Setup a basic `config.yaml` file and put the token in it.
+   Here is a mimumal example config:
```
Comment (drewpearce) [inline docs/BASICS.md]: s/minumal/minimal

File: docs/BASICS.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    path: Legobot.Connectors.Slack.Slack
+    kwargs:
+      token: TOKEN-SECRET-GOES-HERE-1337
+helpEnabled: true
+legos:
+  TestingConnector:
+    enabled: true
+    path: legos.TestingConnector
+   ```
+6. Setup a basic `chatbot.py` file and put the token in it.
```
Comment (drewpearce) [inline docs/BASICS.md:29]: If we merge your other PR for __main__.py you'll want to update this accordingly

File: setup.cfg [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 name = Legobot
 version = 1.3.6
-author = Kevin McCabe, Bren Briggs, and Drew Bronson, Drew Pearce
+author = Kevin McCabe, Bren Briggs, Drew Bronson, and  Drew Pearce
```
Comment (drewpearce) [inline setup.cfg:4]: 😹

**Review by drewpearce — CHANGES_REQUESTED** [review]

File: docs/BASICS.md (hunk 2 above)
Comment (ispringle) [reply docs/BASICS.md:29]: I was planning on updating the docs in that PR.

File: docs/BASICS.md (hunk 1 above)
Comment (ispringle) [reply docs/BASICS.md]: ✔️

**Review by drewpearce — APPROVED** [review]

**Pull request closed without merging**